vrk-kpa/suomidigi
# SUOPA-142 Attachments

To test: 
- `make fresh`
- Edit a landing page or article
- Add an Attachments paragraph to the page
   - Within the attachments paragraph, add few new Attachment and External attachment media items
- Save the page and check that the attachments looks as they do in the proto: https://ixonos.invisionapp.com/share/6JQRIITSPYK#/screens/358946005_Desktop_Kehittaminen-Menetelma-t--artikkeli
